source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/organizations/common/form.tsx

```diff
@@ -0,0 +1,279 @@
+        buildEngineUrl,
+        buildEngineApiAccessToken,
+        logoUrl,
+        publicByDefault,
+        owner
+      };
+
+    } else {
+      this.state = {
+        name: '',
+        websiteUrl: '',
+        buildEngineUrl: '',
+        buildEngineApiAccessToken: '',
+        logoUrl: '',
+        publicByDefault: false,
+        owner: null,
+        ownerError: ''
+      };
+    }
+
+  }
+
+  isValidForm = () => {
+    const { owner } = this.state;
+    if (!owner) {
+      this.setState({ ownerError: 'Cannot be empty' });
+    } else {
+      this.setState({ ownerError: '' });
+    }
+    return owner;
+  }
+
+  submit = async (e) => {
+    e.preventDefault();
+
+    const { onSubmit } = this.props;
+
+    if (this.isValidForm()) {
+      onSubmit(this.state, this.cleanForm);
+    }
+  }
+
+  cleanForm = () => {
+    this.setState({
+      name: '',
+      websiteUrl: '',
+      buildEngineUrl: '',
+      buildEngineApiAccessToken: '',
+      logoUrl: ''
+    });
+  }
+
+  cancel = (e) => {
+    e.preventDefault();
+    const { onCancel } = this.props;
+    onCancel();
+  }
+
+  ownerSelection = user => e => {
+    this.setState({
+      owner: user
+    });
+  }
+
+  render() {
+    const { mut, toggle } = this;
+
+    const {
+      name, websiteUrl, buildEngineUrl,
+      buildEngineApiAccessToken,
+      logoUrl, publicByDefault,
+      owner, ownerError
+    } = this.state;
+
+    const { t, users, organization } = this.props;
+
+    const { name: OwnerName } = attributesFor(owner);
+
+    return (
+      <>
+        <h2>
+          {
+            t(organization ?
+              'admin.settings.organizations.edit' :
+              'admin.settings.organizations.add'
+            )
+          }
+        </h2>
+        <div className='flex w-60'>
+          <form data-test-form className='ui form flex-grow'>
+
+            <div className='field m-b-xl'>
+              <label>{t('admin.settings.organizations.name')}</label>
+              <input
+                data-test-org-name
+                type='text'
+                value={name || ''}
+                onChange={mut('name')} />
+            </div>
+
+            <div className='field m-b-xl'>
+              <label>{t('admin.settings.organizations.owner')}</label>
+              <div className='w-100 thin-bottom-border'>
+                <Dropdown
+                  className='w-100 no-borders'
+                  data-test-org-owner
+                  text={OwnerName}
+                >
+                  <Dropdown.Menu>
+                    {
+                      users.map((u, i) => {
+                        const { name: fullName } = attributesFor(u);
+                        return <Dropdown.Item key={i} text={fullName} onClick={this.ownerSelection(u)} />;
+                      })
+                    }
+                  </Dropdown.Menu>
+
+                </Dropdown>
+              </div>
+              <div className='error'>{ownerError}</div>
```

Comment:
We have an error component in, `import { ErrorMessage } from '@ui/components/errors'`; which will take care of hiding itself when there are no errors